internal/services/appservice/helpers/function_app_slot_schema.go

```diff
@@ -0,0 +1,1267 @@
+						string(web.FtpsStateAllAllowed),
+						string(web.FtpsStateDisabled),
+						string(web.FtpsStateFtpsOnly),
+					}, false),
+					Description: "State of FTP / FTPS service for this function app. Possible values include: `AllAllowed`, `FtpsOnly` and `Disabled`. Defaults to `Disabled`.",
+				},
+
+				"health_check_path": {
+					Type:        pluginsdk.TypeString,
+					Optional:    true,
+					Description: "The path to be checked for this function app health.",
+				},
+
+				"health_check_eviction_time_in_min": { // NOTE: Will evict the only node in single node configurations.
+					Type:         pluginsdk.TypeInt,
+					Optional:     true,
+					Computed:     true,
+					ValidateFunc: validation.IntBetween(2, 10),
+					Description:  "The amount of time in minutes that a node is unhealthy before being removed from the load balancer. Possible values are between `2` and `10`. Defaults to `10`. Only valid in conjunction with `health_check_path`",
+				},
+
+				"worker_count": {
+					Type:         pluginsdk.TypeInt,
+					Optional:     true,
+					Computed:     true,
+					ValidateFunc: validation.IntBetween(1, 100),
+					Description:  "The number of Workers for this Linux Function App.",
+				},
+
+				"minimum_tls_version": {
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+					Default:  string(web.SupportedTLSVersionsOneFullStopTwo),
+					ValidateFunc: validation.StringInSlice([]string{
+						string(web.SupportedTLSVersionsOneFullStopZero),
+						string(web.SupportedTLSVersionsOneFullStopOne),
+						string(web.SupportedTLSVersionsOneFullStopTwo),
+					}, false),
+					Description: "The configures the minimum version of TLS required for SSL requests. Possible values include: `1.0`, `1.1`, and  `1.2`. Defaults to `1.2`.",
+				},
+
+				"scm_minimum_tls_version": {
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+					Default:  string(web.SupportedTLSVersionsOneFullStopTwo),
+					ValidateFunc: validation.StringInSlice([]string{
+						string(web.SupportedTLSVersionsOneFullStopZero),
+						string(web.SupportedTLSVersionsOneFullStopOne),
+						string(web.SupportedTLSVersionsOneFullStopTwo),
+					}, false),
+					Description: "Configures the minimum version of TLS required for SSL requests to the SCM site Possible values include: `1.0`, `1.1`, and  `1.2`. Defaults to `1.2`.",
+				},
+
+				"cors": CorsSettingsSchema(),
+
+				"vnet_route_all_enabled": {
+					Type:        pluginsdk.TypeBool,
+					Optional:    true,
+					Default:     false,
+					Description: "Should all outbound traffic to have Virtual Network Security Groups and User Defined Routes applied? Defaults to `false`.",
+				},
+
+				"detailed_error_logging_enabled": {
+					Type:        pluginsdk.TypeBool,
+					Computed:    true,
+					Description: "Is detailed error logging enabled",
+				},
+
+				"linux_fx_version": {
+					Type:        pluginsdk.TypeString,
+					Computed:    true,
```

Comment:
And looking to confirm if we can't set this

Reply:
as per `windows_fx_version` above